capstone/src/main/java/com/google/step/servlets/RestaurantServlet.java

```diff
@@ -88,8 +90,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         new Restaurant(null, name, geoPoint, story, cuisineList, phone, website, status);
 
     restaurantClient.putRestaurant(restaurant, email);
-
-    response.sendRedirect("/index.html");
```

Comment:
Not a change just a question, so we will send the redirect link as part of the Json I'm guessing?

Reply:
I thought we were redirecting on the frontend instead? Like checking for an error after submitting and if it's clear then redirect with javascript. We can discuss which approach is better though!

Reply:
Alright so its all done through the frontend! Makes sense.